### PR TITLE
UNI-494 throws error on files with less than 2 rows and no valid data row

### PR DESCRIPTION
**Pull request by andrewmalta13**
Fails loudly rather than quietly when a user uploads a file with less than 2 rows an neither of them
are valid data rows. Updated tests to reflect this change.

Please Review:
@marionleborgne

### CODE REVIEW

File: unicorn/app/main/FileService.js [hunk 1]
```diff
@@ -287,6 +287,15 @@ export class FileService {
         headers = values;
         offset++;
       });
+
+    stream.on('end', () => {
+      // We reached the end of the csv and we read either 0 or 1 rows (each)
+      // of which were not valid data rows.
+      if (offset < 2) {
+        callback('Csv has no valid data rows to read');
```
Comment (marionleborgne) [inline unicorn/app/main/FileService.js]: Replace message by  `The CSV file does not have any valid data (see supported formats)`  with `supported formats` a hyperlink to `http://numenta.com/htm-studio#get-started`

**Comment by marionleborgne** [comment]
👍 LGTM after you address the comment

**Comment by andrewmalta13** [comment]
updated @marionleborgne

**Comment by marionleborgne** [comment]
👍

**Pull request merged**